indico/modules/rb/models/reservation_occurrences_test.py

```diff
@@ -258,7 +258,7 @@ def test_cancel(smtp, create_reservation, dummy_user, silent, reason):
     else:
         assert occurrence.reservation.edit_logs.count() == 1
         assert occurrence.reservation.edit_logs[0].user_name == dummy_user.full_name
-        extract_emails(smtp, count=2, regex=True, subject=r'Booking cancelled on .+ \(SINGLE OCCURRENCE\)')
+        extract_emails(smtp, count=2, regex=True, subject=r'Booking cancelled .+ \(SINGLE OCCURRENCE\)')
```

Comment:
What do you think about using a subject like `Booking cancelled: XXX` instead of `Booking cancelled XXX` (the latter sounds like the booking cancelled XXX, and I hope neither our meeting rooms nor their bookings ever become sentient :D)

Reply:
I added some [parenthesis](https://github.com/indico/indico/pull/4511/files#diff-8d9f968452beacfa32590ba89f5283b0R51).

Reply:
you're right, i was just looking at the `subject=...` when creating the notifications